### ISSUE TITLE
AWS token not working

### ISSUE THREAD

**Issue by mgrove** [edgi-govdata-archiving/archivers.space#73]
When I try to use the AWS token to upload data for 9568D317-07C4-4FF6-92BB-03DA4D21D239, it says "access denied" when I try to run the aws cp step. I have tried several times a few days apart so it would generate a new AWS token, and neither time has worked. The zip file is about 15 GB, and I'm using Ubuntu.

**Comment by annafil** [comment]
I seem to have the same problem. I got it working only once, the very first time. My upload completed, but I guess the token timed out because I couldn't link the uploaded file to my UUID using the check for file button (CF630B52-CEC2-4996-B06E-3C348A92C47F). Subsequent attempts involved generating new keys and then resulted in what I think is the same access denied error: 

`upload failed: An error occurred (AccessDenied) when calling the CreateMultipartUpload operation: Access Denied`

I am on a Mac, and tried unsetting bash environment variables then setting them back to the new keys.  I also tried different networks, but not yet several days apart, so I'm on my second set of auth tokens. 

@mgrove did you ever figure this out?

**Comment by mgrove** [comment]
No, I never figured it out. I'm considering checking in the URL and moving on soon, but since I already have the data, I'm reluctant to risk someone else collecting the same data. I wonder if anyone has gotten the AWS key to work.

**Comment by dcwalk** [comment]
@b5 or @kmcculloch could you chime in on this?
